packages/ckeditor5-list/docs/features/lists.md

```diff
@@ -4,7 +4,7 @@ category: features
 
 # Lists
 
-The {@link module:list/list~List list} feature allows creating ordered (numbered) and unordered (bulleted) lists in the editor.
+The {@link module:list/list~List list} feature allows creating ordered (numbered) and unordered (bulleted) lists in the editor aimed at better structuring and presenting specific content such as enumerating elements, creating *tables of content* or {@link features/todo-lists to-do lists}.
```

Comment:
Try to avoid such long sentences, the intros are becoming too complex to read which does not contribute to the promotion of the features. Also the TOC formatting can be skipped.